index.html

```diff
@@ -146,3 +146,4 @@
     <script src="js/index.js"></script>
```

Comment:
The script tag was not supposed to be there

Reply:
<ul>
          <li></li>
          <li></li>
          <li></li>
          <li></li>
          <li></li>
        </ul>
why all the empty li add some or delete them id there's no need

Reply:
Kindly add a reason for each required changes :wink: . This will help the student understand and make it easy to implement the required task. :+1:

Reply:
- [ ] Please implement the suggestions of the previous code reviewer; this project aims to learn how to do a code review and learn HTML and CSS simultaneously. Kindly make your suggestions /feedback clear and meaningful, so students understand what you said and why you need a change. 

***For example***, " Kindly change the title for the project with the appropriate one; the title does not match your project. Please fix :wrench: "